### DISCUSSION TITLE
Welcome to Villain Discussions!

### DISCUSSION THREAD

**Discussion by btzr-io** [btzr-io/Villain#922] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GigiBL** [comment]
hi im marjorie bell i just wanted to say how much i love reading comic books in Villian it works amazingly well for everything totally recommend! ty

**Reply by btzr-io** [reply]
Thank you!
